Containerfile.redis

```diff
@@ -1,39 +1,18 @@
 # Use Red Hat Universal Base Image (UBI) 9 as the base image
 FROM registry.access.redhat.com/ubi9/ubi-minimal
 
-# Install necessary dependencies (gcc, make, etc.) for building Redis from source
-RUN microdnf install -y gcc make wget tar gzip vim openssl openssl-devel tcl && \
-    microdnf clean all
-
-# Set the Redis version
-ARG REDIS_VERSION=7.2.4
-
-# Download and extract Redis source code
-RUN mkdir -p /etc/redis && \
-    wget https://download.redis.io/releases/redis-${REDIS_VERSION}.tar.gz && \
-    tar xzf redis-${REDIS_VERSION}.tar.gz -C /etc/redis --strip-components=1 && \
-    rm redis-${REDIS_VERSION}.tar.gz && \
-    chmod 777 /var/run
-
-# Change working directory to the extracted Redis source code directory
+# Install necessary dependencies for downloading and installing Redis
+RUN microdnf install -y wget yum-utils logrotate systemd && \
+    wget https://mirror.stream.centos.org/9-stream/AppStream/x86_64/os/Packages/redis-6.2.7-1.el9.x86_64.rpm && \
+    rpm -i redis-6.2.7-1.el9.x86_64.rpm && \
+    rm -f redis-6.2.7-1.el9.x86_64.rpm && \
```

Comment:
if we're deliberately switching to redis6 (and i'm not sure why we would, since rhel offers redis 7 and v7 will be in support longer), it can definitely be installed on centos using a centos base image, with just a dnf install.

which we should probably doing regardless (using a centos base image, if we're going to add centos content..i don't see a good reason not to):

https://quay.io/repository/centos/centos?tab=tags